nodejs/esbuild-express/commands/run

```diff
@@ -6,6 +6,7 @@ echo "Install, link and build integration"
 (
   cd /integration
   script/setup
+  git checkout 2-4-stable
```

Comment:
Best thing to do is to move it one line up so we're sure to also run the right version's `script/setup` script. Same applies to all other files you changed.

Reply:
Ah, that makes sense, thank you!